main.py

```
@@ -344,79 +344,114 @@ async def whatsapp_webhook(Body: str = Form(...), From: str = Form(...)):
-    
+    # 3. Local persistent PEM file (dev/staging only)
     if os.path.exists(PRIVATE_KEY_PATH):
-        with open(PRIVATE_KEY_PATH, "rb") as f:
-            return serialization.load_pem_private_key(f.read(), password=None)
-    
+        try:
+            with open(PRIVATE_KEY_PATH, "rb") as f:
+                _cached_private_key = serialization.load_pem_private_key(f.read(), password=None)
+            print(f"Key Management: Loaded existing local key from {PRIVATE_KEY_PATH}")
+            return _cached_private_key
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)